config/environments/development.rb

```diff
@@ -42,4 +42,16 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  config.paperclip_defaults = {
+    storage: :s3,
+    s3_credentials: {
+      access_key_id: "AKIAIXRHXAZSSNZC4JVQ",
+      bucket: "grants.hifinfo.org",
+      secret_access_key: "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",
```

Comment:
I recommend removing this and relying on the default behavior, which is local storage in `#{Rails.root}/public/system`.  Using S3 in development or test could cause developers working on different machines to overwrite or delete each other's data.